packages/fxa-content-server/app/scripts/templates/settings.mustache

```diff
@@ -1,4 +1,9 @@
 <div id="fxa-settings-header-wrapper">
+  {{#ccExpired}}
+    <div class="cc-expired-alert" role="alert">
+      {{#unsafeTranslate}}Your credit card has expired. Please update it <a %(ccExpiredLinkAttrs)s>here</a>.{{/unsafeTranslate}}
```

Comment:
I'm really surprised [ccExpiredLinkAttrs isn't being flagged as an invalid variable name](https://github.com/mozilla/fxa/blob/c2e696500db9715eca8a42660c60c5a016335050/packages/fxa-content-server/app/scripts/views/base.js#L503). Variable names within `unsafeTranslate` are supposed to [require an `escaped` prefix](https://github.com/mozilla/fxa/blob/c2e696500db9715eca8a42660c60c5a016335050/packages/fxa-content-server/app/scripts/lib/strings.js#L12) to remind the developer to properly escape them (comment left below about that).

Reply:
I'll add another test case. Looks like I just need to replace my mock context with a backbone model and then call `renderTemplate` to trigger it...